src/containers/Plot/index.tsx

```diff
@@ -46,8 +57,10 @@ class Plot extends React.Component<PlotProps, {}> {
         }
     }
 
-    shouldComponentUpdate() {
-        return this.props.plotConfig.shouldRenderTimeIndicator;
+    shouldComponentUpdate(nextProps: PlotProps, nextState: PlotState) {
```

Comment:
sanity check: that this doesn't prevent plots without time indicators from ever updating, does it??

Reply:
in other words.. if a plot doesn't have a time indicator, does it still get drawn properly and update when it should?

Reply:
yeah, I had that same reaction to this, but this component re-mounts when there are new plots, so the non time plots get drawn once on mount and never update.

Reply:
What Megan said -- non-time plots have no reason to update after mounting, at least not at this time. I just verified that built-in plotly user interactions (like zooming in) still works properly.